moshi/src/main/java/com/squareup/moshi/Json.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright (C) 2015 Square, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.squareup.moshi;
+
+import java.lang.annotation.Documented;
+import java.lang.annotation.Retention;
+import java.lang.annotation.Target;
+
+import static java.lang.annotation.ElementType.FIELD;
+import static java.lang.annotation.RetentionPolicy.RUNTIME;
+
+/** Customizes how a field is encoded as JSON. */
+@Target(FIELD)
+@Retention(RUNTIME)
+@Documented
+public @interface Json {
+  String name();
```

Comment:
I was thinking we might eventually offer other options, like `required = true` or `serializeNull = true` or whatever. When that happens it's weird to have this:

```
  @Json(value = "phone number", required = true);
  String phoneNumber;
```

... since it's not a value.